### ISSUE TITLE
Scaladoc comments in derived objects

### ISSUE THREAD

**Issue by dsilvasc** [sangria-graphql/sangria#297]
Would it be possible to support scaladoc comments in classes, traits, fields, and methods for derived objects as an alternative to GraphQLDescription annotations?

**Comment by OlegIlyenko** [comment]
No, I afraid it would not be possible since reflection/macro API does not expose scaladoc information.
